core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -299,7 +299,7 @@
   // Deprecated. It will be removed in 2.0.0.
   USER_UFS_FILE_WRITER_CLASS(Name.USER_UFS_FILE_WRITER_CLASS,
       "alluxio.client.netty.NettyUnderFileSystemFileWriter"),
-  USER_LOCAL_READER_PACKET_SIZE_BYTES(Name.USER_LOCAL_READER_PACKET_SIZE_BYTES, "64KB"),
+  USER_LOCAL_READER_PACKET_SIZE_BYTES(Name.USER_LOCAL_READER_PACKET_SIZE_BYTES, "1MB"),
```

Comment:
This is increased to achieve similar performance as the legacy one when reading from  local as the impl is similar